### ISSUE TITLE
Failure if work dir has space in path

### ISSUE THREAD

**Issue by seandavi** [nextflow-io/nextflow#86]
```bash
  echo 'abc' > trimmed.fq

Command exit status:
  1

Command output:
  (empty)

Work dir:
  /gpfs/gsfs2/users/sdavis2/projects/pNET Microarray Data/work/0a/e4c70e1d39acc0d94aff0d5ae6f4b0

Tip: you can replicate the issue by changing to the process work dir and entering the command: 'bash .command.run'

WARN: Killing pending processes (10)
[sdavis2@cn1869 pNET Microarray Data]$ cd work/0a/e4c70e1d39acc0d94aff0d5ae6f4b0
[sdavis2@cn1869 e4c70e1d39acc0d94aff0d5ae6f4b0]$ bash .command.run
touch: cannot touch `Data/work/0a/e4c70e1d39acc0d94aff0d5ae6f4b0/.command.begin': No such file or directory
```

**Comment by pditommaso** [comment]
Thanks for reporting that. As a workaround you can specify a working directory with no blanks using the `NXF_WORK` environment variable or `-w` command line option. For example: 

    nextflow run <script> -w /some/work/path

**Comment by pditommaso** [comment]
Fixed in version `0.16.2`. 

@seandavi Note, if you are are using Docker containers it may fail when using a working directory containing blank characters. See https://github.com/docker/docker/issues/17967
